src/Resources/scaffolds/6.0/reset-password/src/Entity/ResetPasswordRequest.php.tpl

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ResetPasswordRequestRepository;
+use Doctrine\ORM\Mapping as ORM;
+use SymfonyCasts\Bundle\ResetPassword\Model\ResetPasswordRequestInterface;
+use SymfonyCasts\Bundle\ResetPassword\Model\ResetPasswordRequestTrait;
+
+#[ORM\Entity(repositoryClass: ResetPasswordRequestRepository::class)]
+class ResetPasswordRequest implements ResetPasswordRequestInterface
+{
+    use ResetPasswordRequestTrait;
+
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
+    private $id;
+
+    #[ORM\ManyToOne(targetEntity: User::class)]
+    #[ORM\JoinColumn(nullable: false)]
+    private $user;
```

Comment:
```suggestion
    private User $user;
```

Then we don't need `targetEntity` No need for "nullable" in this case, since we set it in the constructor

Reply:
> Then we don't need targetEntity

Cool, didn't know that.

> No need for "nullable" in this case, since we set it in the constructor

I believe the column is nullable in the db by default. Or does typehinting the non-nullable User on the property set it as non-nullable?

Reply:
Maybe you just meant "no need for `?User`". I also added `onDelete: 'CASCADE'` to the join column.